cmd/entrypoint/main.go

```diff
@@ -64,6 +68,20 @@ func cp(src, dst string) error {
 	return err
 }
 
+func checkForBreakpointOnFailure(e entrypoint.Entrypointer, breakpointExitPostFile string) {
+	if e.BreakpointOnFailure {
+		if waitErr := e.Waiter.Wait(breakpointExitPostFile, false, false); waitErr != nil {
+			log.Println("error occurred while waiting for " + breakpointExitPostFile)
+		}
+		// get exitcode from .breakpointexit
+		exitCode, readErr := e.ReadBreakpointExitCodeFromDisk(breakpointExitPostFile)
+		if readErr != nil {
+			log.Println("error occurred while reading breakpoint exit code : " + readErr.Error())
```

Comment:
If we fail to read the exit code, AIUI the code as currently written will `os.Exit(0)` after logging the error (because `exitCode` defaults to 0). Is that intentional? It might be subtle to readers, and deserve a comment.

Reply:
This is subtle indeed, I added the following comment
```
// if readErr exists, the exitcode with default to 0 as we would like
// to encourage to continue running the next steps in the taskRun
```